runners/spark/src/main/java/org/apache/beam/runners/spark/translation/SparkKeyedCombineFn.java

```diff
@@ -0,0 +1,87 @@
+package org.apache.beam.runners.spark.translation;
```

Comment:
Looks like this file is missing Apache license header, so RAT is complaining in the build.